packages/wdio-cucumber-framework/src/utils.js

```diff
@@ -147,38 +147,6 @@ export function compareScenarioLineWithSourceLine(scenario, sourceLocation) {
     return scenario.location.line === sourceLocation.line
 }
 
-export function getStepFromFeature(feature, pickle, stepIndex, sourceLocation) {
```

Comment:
had to remove this way of getting steps, it didn't at all in some cases.

We should rely on steps received in `testCasePreparedEvent` instead of relying on feature and mixing up steps in a hacky way. Didn't know this before